### PR TITLE
Update to Django 4.2

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
still some unsolved issues, can anybody help (see below)?

### CODE REVIEW

File: src/nsupdate/main/models.py [hunk 1]
```diff
@@ -253,8 +258,8 @@ def __str__(self):
         return u"%s.%s" % (self.name, self.domain.name)
 
     class Meta(object):
-        unique_together = (('name', 'domain'),)
-        index_together = (('name', 'domain'),)
+        constraints = [models.constraints.UniqueConstraint(fields=['name', 'domain'], name='unique_host_domain')]
+        indexes = [models.Index(fields=['name', 'domain'])]
         verbose_name = _('host')
         verbose_name_plural = _('hosts')
```
Comment (ThomasWaldmann) [inline src/nsupdate/main/models.py:264]: something is wrong here. any idea, @nougad ?

*_together are deprecated in django 4.2 / soon.

**Comment by jhass** [comment]
Looks like 4.2.1 came out May 3rd :)

**Comment by ThomasWaldmann** [comment]
@jhass that's great. but someone still has to solve the issues in this PR, see above, see failing tests.

**Comment by ThomasWaldmann** [comment]
superseded by #529.

**Pull request closed without merging**